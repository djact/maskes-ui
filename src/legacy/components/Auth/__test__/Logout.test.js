import React from 'react'
import { shallow } from 'enzyme'
import { Logout } from '..'

describe('Test Logout Component', () => {
    it('should shallow render correctly Logout component', () => {
        const LogoutComponent = shallow(<Logout />)
        expect(LogoutComponent).toMatchSnapshot()
    })
})
