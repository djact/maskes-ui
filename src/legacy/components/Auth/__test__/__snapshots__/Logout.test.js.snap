// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Logout Component should shallow render correctly Logout component 1`] = `
<Redirect
  to="/"
/>
`;
